lib/src/showcase_context_provider.dart

```diff
@@ -0,0 +1,22 @@
+import 'package:flutter/material.dart';
+
+/// This class is used to provide context of Showcase widget to Overlay so
+/// that we can access ShowcaseWidget in widget tree from overlay.
+class ShowcaseContextProvider extends InheritedWidget {
```

Comment:
I think this class isn't needed, showcase context can be accessed using `ShowCaseWidget.of(context)`

Reply:
@ujas-m-simformsolutions  It did not work inside the tooltip action widget that's why created.